docs/setup_wsl.md

```diff
@@ -113,43 +118,75 @@ Enter a new UNIX username:
 
 </div>
 
-## Install CLI tools
-From an Ubuntu terminal, use the `apt` package manager to install a few command line programs.  Linux users will run this same command.
+### Install CLI tools
+From an Ubuntu bash terminal, use the `apt` package manager to install a few command line programs.
 ```console
 $ sudo apt update
 $ sudo apt install g++ make rsync wget git ssh gdb python3 tree
 ```
 {: data-variant="no-line-numbers" }
 
-## Use CLI tools
-Now would be a great time to take a look at our [CLI Tutorial](cli.html).
+### Home Directory
+You'll generally want to store your coding projects in the Linux filesystem, which is separate from your Windows filesystem.
 
-## Pro-tips
+Open an Ubuntu bash terminal and run `cd ~`. This will take you to your Ubuntu home directory. Running `pwd` afterward confirms the location.
 
-### Accessing Linux files from Windows
-To access Linux files from the Windows File Explorer, click the Linux icon.  You may need to scroll down.
+```console
+$ cd ~
+$ pwd
+/home/jjuett
+```
 
-Navigate to your home directory: Linux > Ubuntu > home > awdeorio.  Your username will be different.
+Create an EECS 280 folder by running `mkdir eecs280`. Running `ls` afterward confirms the folder has been created.
 
-<img src="images/wsl080.png" width=768px>
+```console
+$ mkdir eecs280
```

Comment:
I can imagine some students accidentally running this from the wrong folder.

```suggestion
$ mkdir ~/eecs280
```

Reply:
Good call, the absolute path makes it foolproof.